src/jdk.jfr/share/classes/jdk/jfr/internal/JDKEvents.java

```diff
@@ -73,6 +77,8 @@ public final class JDKEvents {
         jdk.internal.event.X509ValidationEvent.class,
         DirectBufferStatisticsEvent.class,
         InitialSecurityPropertyEvent.class,
+        MethodTraceEvent.class,
+        MethodTimingEvent.class
```

Comment:
```suggestion
        MethodTimingEvent.class,
```
for future additions.